### PR TITLE
update: goreleaser

### PR DESCRIPTION
**Pull request by caarlos0**
- [ ] Have you followed the [guidelines for contributing](https://github.com/Homebrew/homebrew-core/blob/master/CONTRIBUTING.md)?
- [x] Have you checked that there aren't other open [pull requests](https://github.com/Homebrew/homebrew-core/pulls) for the same formula update/change?
- [ ] Have you built your formula locally with `brew install --build-from-source <formula>`, where `<formula>` is the name of the formula you're submitting?
- [ ] Does your build pass `brew audit --strict <formula>` (after doing `brew install <formula>`)?

-----

ideally I should also add `"-X main.commit=commit_sha256"` and `"-X main.date=2018-11-08THH:MM:SSZ"` but I don't know how to do it here...

### CODE REVIEW

**Comment by fxcoudert** [comment]
```
==> go build -ldflags -s -w -X main.version=0.93.1 -o /usr/local/Cellar/goreleaser/0.93.1/bin/goreleaser
flag provided but not defined: -w
usage: go build [-o output] [-i] [build flags] [packages]
Run 'go help build' for details.
```

File: Formula/goreleaser.rb [hunk 1]
```diff
@@ -17,8 +17,9 @@ def install
     ENV["GOPATH"] = HOMEBREW_CACHE/"go_cache"
     (buildpath/"src/github.com/goreleaser/goreleaser").install buildpath.children
     cd "src/github.com/goreleaser/goreleaser" do
-      system "go", "build", "-ldflags", "-X main.version=#{version}", "-o",
-             bin/"goreleaser"
+      system "go", "build", 
+             "-ldflags", "-s", "-w", "-X main.version=#{version}",
```
Comment (apjanke) [inline Formula/goreleaser.rb]: I think you meant:
```
         "-ldflags", "-s -w -X main.version=#{version}", 
```

But I don't see why the `-s` and `-w` are necessary. Per the `man ld` page, `-s` is obsolete, and `-w` just suppresses warnings, which I don't think we want. Has something changed between releases to motivate adding `-s -w`?

**Review by apjanke — CHANGES_REQUESTED** [review]

File: Formula/goreleaser.rb (hunk 1 above)
Comment (caarlos0) [reply Formula/goreleaser.rb]: is just what is used by goreleaser build itself... anyway, you're right, its not needed... will undo that..

**Comment by fxcoudert** [comment]
Thanks @caarlos0

**Comment by caarlos0** [comment]
Thanks @fxcoudert !

**Pull request closed without merging**